uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -330,6 +331,22 @@ public void setMqttTimeoutSec(int mqttTimeoutSec) {
      */
     @When("I subscribe {string} to {string} with qos {int}")
     public void subscribe(@NonNull String clientDeviceId, @NonNull String topicFilterString, int qos) {
+        subscribe(clientDeviceId, topicFilterString, qos, 2);
```

Comment:
Can we use "Mqtt5RetainHandling.MQTT5_RETAIN_DO_NOT_SEND_AT_SUBSCRIPTION" instead of magic number "2"?